openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -782,18 +781,12 @@ port_INLINE bool ieee154e_processIEs(OpenQueueEntry_t* pkt, uint16_t* lenIE) {
             // at this point, ASN and frame length are known
             // the current slotoffset can be inferred
             ieee154e_syncSlotOffset();
-            schedule_syncSlotOffset(ieee154e_vars.slotOffset);
-            ieee154e_vars.nextActiveSlotOffset = schedule_getNextActiveSlotOffset();
-            /* 
-            infer the asnOffset based on the fact that
-            ieee154e_vars.freq = 11 + (asnOffset + channelOffset)%16 
-            */
-            for (i=0;i<16;i++){
-                if ((ieee154e_vars.freq - 11)==ieee154e_vars.chTemplate[i]){
-                    break;
-                }
+            if (schedule_syncSlotOffset(ieee154e_vars.slotOffset)==TRUE) {
+               ieee154e_vars.nextActiveSlotOffset = schedule_getNextActiveSlotOffset();
+            } else {
+               ieee154e_vars.isSpouriousEB = TRUE;

```

Comment:
The name `isSpouriousEB` is a bit misleading. Is there not a more comprehensive name?